services/sso/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 All notable changes to this project will be documented in this file.
 # Changelog
 
+## [1.20.1] - 2024-04-18
```

Comment:
This is not a patch change, extends the functionality rather than merely fixing a bug.

Reply:
fixed